simpson.cpp

```diff
@@ -0,0 +1,45 @@
+#include<stdio.h>
+#include<conio.h>
+float f(float x)
+{
+    return(1/(1+x));
+}
+void main()
+{
+    int i,n;
+    float x0,xn,h,y[20],so,se,ans,x[20];
```

Comment:
Declare these variables when they are first used, not at the top of the function.